src/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
nEnds":0,"TestDBO::setConfigDateTimeSetsVotingBegins":0,"TestDBO::setConfigDateTimeSetsVotingEnds":0,"TestDBO::setConfigDateTimeThrowsExceptionForUnknown":0,"TestDBO::setConfigPrepBoFWorksIfNotAlreadyConfigured":0,"TestDBO::setConfigPrepBoFWorksIfConfigured":0,"TestDBO::setLocationNamesSetsLocationsSequentially":0,"TestDBO::setRoundNamesSetsRoundsSequentially":0,"TestDBO::switchBookingsSwitchesOnlyWhatItShould":0,"TestDBO::updateWorkshopUpdatesOnlySpecifiedWorkshop":0,"TestDBO::updateWorkshopHandlesEmptyPublishedValue":0,"TestDBO::validateLocationsWhenValid":0,"TestDBO::validateLocationsWhenInvalid":0,"TestDBO::validateRoundsWhenValid":0,"TestDBO::validateRoundsWhenInvalid":0,"TestLogger::clearLogReturnsEmpty":0,"TestLogger::getLogReturnsEmptyIfNothingLogged":0,"TestLogger::getLogReturnsLoggedMessage":0,"TestLogger::getLogReturnsEmptyAfterGetLog":0,"TestLogger::logAppendsWithNewlines":0,"TestLogger::logBookWorkshop":0.001,"TestLogger::logSwitchedWorkshops":0,"TestModeration::moderateAddsFacilitator":0.001,"TestModeration::moderateDeletesWorkshop":0.001,"TestModeration::moderateMergesWorkshops":0.002,"TestModeration::moderateUpdatesWorkshops":0.002,"TestModeration::showModerationViewThrowsExceptionForNonAdmin":0.002,"TestModeration::showModerationViewRendersPage":0.001,"TestNomination::nominateInvokesDBONominateWithProperArgs":0.001,"TestNomination::nominateReturnsErrorForEmptyTitle":0,"TestNomination::nominateReturnsErrorForNullTitle":0,"TestNomination::nominateReturnsErrorForEmptyDescription":0,"TestNomination::nominateReturnsErrorForNullDescription":0,"TestNomination::nominateReturnsErrorForEmptyTitleAndDescription":0,"TestNomination::nominateShowsErrorIfDBONominateThrowsException":0.002,"TestProjector::showProjectorViewReturnsEmptyDataForBadStage":0.001,"TestProjector::showProjectorViewReturnsNoVoteAndLeaderForNominatingStage":0.001,"TestProjector::showProjectorViewShowsFinishedStageForFinished":0.002,"TestProjector::showProjectorViewShowsVotingStageForLocked":0.002,"TestProjector::showProjectorViewShowsVotingStageForVoting":0.001,"TestResults::calculateResultsWithNoPrepBoF":0.358,"TestResults::calculateResultsWithPrepBofAt01":0.27,"TestResults::calculateResultsWhenNoConflictsEnoughWorkshops34":0.269,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops34":0.274,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops25":0.191,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops52":0.207,"TestResults::calculateResultsConflictsWhenConflictsNotEnoughWorkshops34":0.237,"TestResults::calculateResultsWithMaxConflicts":0.277,"TestResults::calculateResultsThrowsExceptionIfInvalidLocations":0.002,"TestResults::calculateResultsThrowsExceptionIfInvalidRounds":0.001,"TestResults::calculateResultsThrowsExceptionIfTooFewLocations":0,"TestTimezones::testTimezonesList":0.013,"TestTimezones::testTimezonesDateTimeSetTimezone":0,"TestTimezones::testTimezonesParseDateTimeWithTimezone":0,"TestTimezones::testParseAndUtc":0}}
```

Comment:
We probably want to exclude cache files like this...

Reply:
Sure thing. I can update the .gitignore to accommodate.